cumulus_library/cli.py

```diff
@@ -15,23 +15,26 @@
 from cumulus_library.study_parser import StudyManifestParser
 
 
+# ** Don't delete! **
+# This class isn't used in the rest of the code,
+# but it is used manually as a quick & dirty alternative to the CLI.
 class CumulusEnv:  # pylint: disable=too-few-public-methods
     """
     Wrapper for Cumulus Environment vars.
     Simplifies connections to StudyBuilder without requiring CLI parsing.
     """
 
     def __init__(self):
-        self.bucket = os.environ.get("CUMULUS_LIBRARY_S3")
+        self.s3_output = os.environ.get("CUMULUS_LIBRARY_S3_OUTPUT")
```

Comment:
how would you feel about calling this `s3_staging` instead?

the docs for pyathena leave something to be desired in this regard - but, assuming you're testing in HMS AWS, it might be a bucket permissioning issue on the default athena cache location??

Reply:
The docs for `pyathena` are terrible. I've been looking at code for most of my questions, and even then it's not super helpful.

tl;dr; sure `staging` is fine.

But I have some thoughts:
- What does `pyathena` do with this folder? Is it really just the "Query result location" for the workgroup? Like, is it dropping query results there, via Athena queries? Or some other unnamed artifacts...?
- The user would be familiar with the phrase "query result location" or "output location" (or at least, they theoretically could be -- it's what the fields are called in the web UI and CloudFormation for workgroups respectively) -- do they know what is being asked of them when we say `S3 staging dir`? I don't know what PyAthena is asking of us, so I doubt a partner would.
- One nice thing about `staging` is it avoids any confusion with the "ETL output dir" that the user previously set up
- I'm annoyed that PyAthena wants this at all. The code seems to only require workgroup or staging (which is perfectly consistent with "we'll grab the staging dir from the workgroup" -- dunno why it failed for me)

Reply:
Renamed to `--s3-staging` and `CUMULUS_LIBRARY_S3_STAGING`, but I still worry that users will have no idea what that is.

Reply:
OK we talked in person - and we both fixed our separate user-error errors. So now it looks like we can delete this option entirely. I've updated the PR